src/components/side_panel/chart/geo_chart_panel/geo_chart_config_panel.ts

```diff
@@ -7,14 +6,7 @@ export class GeoChartConfigPanel extends GenericChartConfigPanel {
   static components = { ...GenericChartConfigPanel.components, GeoChartRegionSelectSection };
 
   get dataRanges() {
-    return this.getDataSeriesRanges().slice(0, 1);
-  }
-
-  onDataSeriesConfirmed() {
-    this.dataSets = spreadRange(this.env.model.getters, this.dataSets).slice(0, 1);
-    this.state.datasetDispatchResult = this.props.updateChart(this.props.figureId, {
-      dataSets: this.dataSets,
-    });
+    return this.getDataSeriesRanges().map((ds, i) => ({ ...ds, ignored: i > 0 }));
```

Comment:
So did you decide to have isIgnored being handled in the chart definition (w/ a migration) or in the side panel directly ? You're doing both ATM